src/index.js

```diff
@@ -1,2 +1,15 @@
 import _ from 'lodash'; // eslint-disable-line
-import './style.css';
\ No newline at end of file
+import './style.css';
+import fetchAPIData from './Modules/Get data.js';
+import postAPIData from './Modules/Post data.js';
+
+document.querySelector('DOMContentLoaded', fetchAPIData());
+
```

Comment:
While you did an awesome job working on the project requirements I noticed you disables eslint. It is against microverse policy to disable linters because linters help us write clean code. It would be nice if you enabled eslint.